src/Application.php

```diff
@@ -21,6 +21,7 @@ public function __construct($name)
     {
         parent::__construct($name, 1337);
         $this->name = $name;
+        $this->config = array();
```

Comment:
Only if no-config files are loaded, will fix this into something more suitable for the context